docs/source/model_design/general_reference_material/bliss/index.rst

```diff
@@ -485,62 +485,87 @@ More information on the methods can be found in the `demographics methods append
 4.0 Results
 +++++++++++
 
-As mentioned above, before beginning our research, we should determine the variables
-of interest: the facets of gender and/or sex that we expect to be most directly associated with
-our outcome.
-If it isn't possible to narrow this down or distinguish what might be important, we can consider
-all gender- and sex-related factors to be variables of interest.
+4.1 Goals when Reporting Results
+--------------------------------
 
-When we present our findings, we have two goals:
+When we present our findings, we have two primary goals:
 
-* Accurately convey what associations we are looking for, and what the causal structures might be.
+* Accurately convey the variables of interest, and what the causal structures might be.
   We should strive to be as precise as possible, and not rely on "common sense"
   to indicate which gender- and sex-related factors are related to a health outcome.
   Despite prevailing belief that sex affects gender but not vice versa, there is
   strong evidence for causal influence in both directions. [Ritz_and_Greaves_2022]_ [Springer_2012]_
-  Even when we do think that a sex-related factor is most proximal to our outcome,
+  Even when we do think that a sex-related factor is most causally related to our outcome,
   we should still mention that some of its impact could be mediated through
   gender factors and that it is difficult to disentangle these.
 * Highlight data limitations around sex and gender measures.
   We don't want our inclusive language to give the impression that we are able to make accurate
-  conclusions about sexual and gender minority populations when the data prevents us from doing this.
+  conclusions about sexual and gender minority populations when the data prevent us from doing this.
```

Comment:
```suggestion
  conclusions about SGM populations when the data prevent us from doing this.
```
we defined this acronym in the glossary, so could use shorthand version? especially if we use this acronym at other points in the guide